canonical/vanilla-framework
# Remove discourse module and related routes

## Done

Remove old discourse based routes

Fixes https://warthogs.atlassian.net/browse/WD-12508

## QA

- Open https://vanilla-framework-5239.demos.haus/design
    - it should 404
    - Compare with https://vanillaframework.io/design
- Open https://vanilla-framework-5239.demos.haus/design/patterns/segmented-control
    - it should 404
    - compare https://vanillaframework.io/design/patterns/segmented-control 

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
